scripts/server.py

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env python
+
```

Comment:
Wonder if we should keep the name for this as `server` and rename the Docker script to something like `docker-server` or something to make them more distinct?

Reply:
Follow up: we discussed this a bit in Slack and if it were possible I think it'd be nice for these scripts to be able to execute either via `scripts/server` (for environments which have Python 2.7) or via `docker-compose exec app scripts/server` (for environments which have the Dockerized Python runtime).

I believe that since we can assume that `#!/usr/bin/env python` will be set in both cases, these can drop the `.py` file extensions.

In general that should save from having two different sets of scripts -- and we can just add a note to the README to explain how to prefix things with `docker-compose exec` for environments with Docker.

Reply:
👍

Reply:
I'm not sure one can rely on `#!/usr/bin/env python` working on windows.  Perhaps using git shell?  Is the plan to require that?

Reply:
I think we should rely on using the command prompt, at least for the sake of keeping the build tool requirements simple.

Reply:
That makes sense. Ideally it seems like these scripts should be runnable both via

```
python scripts/server
```

and via

```
docker-compose exec app scripts/server
```

and each script's internals can be indifferent to how it's being run.

Reply:
Made this https://gist.github.com/kellyi/45d4defe3a154ea8f0b71c3af9cfc90b as an example of how we might have the scripts work in both environments. It looks like we'll need to use 

```
docker-compose run app python scripts/server.py
```

or whatever script instead of `exec`. I'm not sure how to get the requirements.txt installed libraries to be installed in the container without a Dockerfile; for Node applications we often mount the `node_volumes` dir as a volume in docker-compose. Does `pip install` have a similarly discrete dir for its libraries?

Reply:
I believe we may be able to have `pip install` run as the declared `command` in the docker-compose file for the app service in order to have it ensure that that happens when the container is brought up.